### PR TITLE
Justify tables

### PR DESCRIPTION
**Pull request by milljm**
Switch to using DIV elements instead of TABLE elements so that we can have more control over the style.
Fix some markdown attributes.
Update moose environment package versions.

Refs #7892

### CODE REVIEW

**Comment by milljm** [comment]
fixing a few CSS options...

**Comment by milljm** [comment]
there is one cosmetic issue I found:  

http://mooseframework.org/docs/PRs/7949/site/content/framework/systems/Functions/ImageFunction/

Somehow, the header bar is not rendering below the Developer Links. No big deal (in firefox). However in Chrome, it seems to force the Description label to collide with the Information label... I've fought it for a while before giving up.

**Comment by aeslaughter** [comment]
The cosmetic issue can be fixed by changing the markdown for that file and move the "devel" link box. However, this page shows another issue. The "Required Params" information should not be printing out because there are not parameters, would you look into that?

**Comment by milljm** [comment]
@aeslaughter yep, easy fix....

File: docs/packages.yml [hunk 1]
```diff
@@ -5,21 +5,21 @@
...
 ubuntu16.04:
-    name: moose-environment-5_Ubuntu-16.04_x86_64.deb
+    name: moose-environment-6_Ubuntu-16.04_x86_64.deb
 mint17.1:
     name: moose-environment_mint-1.1-8.x86_64.deb
 mint17.3:
     name: moose-environment-4_LinuxMint-17.3_x86_64.deb
 mint18:
-    name: moose-environment-1_LinuxMint-18_x86_64.deb
+    name: moose-environment-3_LinuxMint-18_x86_64.deb
```
Comment (aeslaughter) [inline docs/packages.yml]: Do you need these in soon?

**Review by aeslaughter — COMMENTED** [review]
I need to update the creation of the corresponding latex tables before this can be merged, so it may be a bit. If you need the package update, you may want to put in a separate commit.

**Comment by milljm** [comment]
Nope. I was just doing it because I felt like it. I am in no hurry. We can correct these when the page goes 'live'

**Comment by aeslaughter** [comment]
@milljm I am closing this in favor of the collapsible solution, please don't get rid of this branch in case we change our minds yet again. Thanks for working on this.

**Pull request closed without merging**